modules/vsphere-problem-detector-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator_{context}"]
+= vSphere Problem Detector Operator
+
+[discrete]
+== Purpose
+
+The {operator-name} checks the cluster for the most common installation and misconfiguration issues.
```

Comment:
You could also add a note that the operator currently checks for storage related issues only. Otherwise the reader might expect cluster-wide checks to be run.